egs/aurora4/asr1/run.sh

```diff
@@ -60,6 +60,9 @@ train_set=train_mix
 train_dev=dev_0330
 train_test=test_0166
 recog_set=dev_0330
+# Most papers are reporting the result on dev_0330. You could also use test_0166 as the recog_set.
```

Comment:
Most papers are reporting the result on dev_0330.
->
Most papers are reporting the result on dev_0330 and we follow to use dev_0330 as our evaluation set.